rental.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'book'
+require_relative 'person'
+
+require 'date'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
There should also be getters and setters for book and person since they can be modified here😉.\
```suggestion
  attr_accessor :date, :book, :person
```